generator/03-syndesis-git-proxy.yml.mustache

```diff
@@ -65,7 +65,12 @@
           readinessProbe:
             tcpSocket:
               port: 8080
-            initialDelaySeconds: 1
+            initialDelaySeconds: 10
+            timeoutSeconds: 5
+          livenessProbe:
```

Comment:
Its the default of 180s then. Though I would leave it like this, but we could set it, too.